crates/bitwarden-core/src/platform/client_platform.rs

```diff
@@ -5,15 +5,54 @@ use super::{
 };
 use crate::{error::Result, Client};
 
+/// A struct containing platform utilities.
 pub struct ClientPlatform<'a> {
     pub(crate) client: &'a Client,
 }
 
 impl<'a> ClientPlatform<'a> {
+    /// Will generate a fingerprint based on the `input`. Given the same `input` This
+    /// method will result in the exact same output.
```

Comment:
suggestion (non-blocking): 
```suggestion
    /// method will always result in the exact same output.
```